packages/greencheck/tests/SitecheckAsTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+require_once __DIR__ . '/TestConfiguration.php';
+
+use phpmock\phpunit\PHPMock;
+use Symfony\Bridge\PhpUnit\DnsMock;
+use TGWF\Greencheck\Sitecheck;
+
+use PHPUnit\Framework\TestCase;
+
+class SitecheckAsTest extends TestCase
+{
+    /**
+     *
+     * @var Sitecheck
+     */
+    protected $sitecheck = null;
+
+    public function setUp() :void
+    {
+        // reset database to known state
+        TestConfiguration::setupDatabase();
+
+        $config     = TestConfiguration::$config;
+        $entityManager   = TestConfiguration::$em;
+
+        // Setup the cache
+        $cache = new Sitecheck\Cache($config);
+        $cache->setCache('default');
+
+        // @todo mock these where needed
+        $greencheckUrlRepository = $entityManager->getRepository("TGWF\Greencheck\Entity\GreencheckUrl");
+        $greencheckIpRepository = $entityManager->getRepository("TGWF\Greencheck\Entity\GreencheckIp");
+        $greencheckAsRepository = $entityManager->getRepository("TGWF\Greencheck\Entity\GreencheckAs");
+        $greencheckTldRepository = $entityManager->getRepository("TGWF\Greencheck\Entity\GreencheckTld");
+
+        // @todo this feels *really* messy. See if this makes sense to move
+        // to TestConfiguration.php
+        global $map;
+        require_once __DIR__ . '/urlMap.php';
+
+
+        $dns = $this->createMock(Sitecheck\DnsFetcher::class);
+        $dns->method('getIpAddressesForUrl')->will($this->returnValueMap($map));
+
+        $this->sitecheck = new Sitecheck($greencheckUrlRepository, $greencheckIpRepository, $greencheckAsRepository, $greencheckTldRepository, $cache, new Sitecheck\Logger($entityManager), 'test', $dns);
+
+        //Cleanup all cache entries to correctly test
+        $cache = $this->sitecheck->getCache();
+        $cache->deleteAll();
+    }
+
+    /**
+     * Check that we can get the as information for an url
+     * https://bitbucket.org/cleanbits/greencheck/issue/1/ip-to-asn-mapping-support
```

Comment:
maybe see if we can get this info into a doc that is included in the repo, as it's not currently publicly accessible.

Reply:
Hey @arendjantetteroo I can't seem to get that access to that - can you grant access to mrchrisadams please?

Reply:
Looks like this info is no longer available as the cleanbits org is no longer there. So probably best to just remove the links everywhere